packages/components-native/src/Form/components/FormCache/FormCache.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useMemo } from "react";
+import { FieldValues, useFormContext, useWatch } from "react-hook-form";
+import omit from "lodash/omit";
+
+interface FormCacheProps<T extends FieldValues> {
+  localCacheId?: string | string[];
+  localCacheKey?: string;
+  localCacheExclude?: string[];
+  setLocalCache: (data: T) => void;
+}
+export function FormCache<T extends FieldValues>({
+  localCacheExclude,
+  localCacheKey,
+  setLocalCache,
+}: FormCacheProps<T>): JSX.Element {
+  const { control, formState } = useFormContext<T>();
+  const { isDirty } = formState;
+
+  const formData = useWatch<T>({ control });
+  const shouldExclude = useMemo(() => {
+    return Array.isArray(localCacheExclude) && localCacheExclude.length > 0;
+  }, [localCacheExclude]);
+
+  useEffect(() => {
+    !localCacheKey &&
+      console.log(
+        "No `localCacheKey` specified on Form. Local copy of form data is now disabled.",
```

Comment:
It's because the `localCacheKey` is not being passed to the mocked form component for testing. We could pass any string to this prop and the logs would not show this. It happens to the tests in jobber-mobile too

Reply:
I seeee - it's a lot less clutter in the terminal with a string for `localCacheKey` in the MockForm which I like.  I'll leave it up to you - if you want to leave it as is that's fine.